frontend/device/android/device.lua

```diff
@@ -244,7 +244,7 @@ function Device:init()
     }
 
     -- disable translation for specific models, where media keys follow gravity, see https://github.com/koreader/koreader/issues/12423
-    if android.prop.model == "moaanmix7" or android.prop.model == "xiaomi_reader" or android.prop.model == "gocolor7" then
+    if android.prop.model == "moaanmix7" or android.prop.model == "xiaomi_reader" or android.prop.model == "gocolor7" or android.prop.model == "go7" then
```

Comment:
I overlooked it last time but it should probably be alphabetical if it's growing (go7 or gocolor7 or moaanmix7 or xiaomi_reader).